dask/values.yaml

```diff
@@ -131,6 +131,76 @@ worker:
       podTargetLabels: [] # PodTargetLabels transfers labels on the Kubernetes Pod onto the target.
       metricRelabelings: [] # MetricRelabelConfigs to apply to samples before ingestion.
 
+additional_worker_groups:
+  - worker:
```

Comment:
I think we should ditch this key. You've nested everything under `.worker` but that feels unnecessary.

You should be able to do something like

```yaml
additional_worker_groups:
 - name: foo
   image:
      repository: "foo/bar"
    ...
 - name: bar
   ...